tests/generation/test_utils.py

```diff
@@ -3196,6 +3196,39 @@ def test_assisted_decoding_in_gpu_cpu(self):
         )
         self.assertTrue(input_length <= out.shape[-1] <= input_length + 20)
 
+    def test_special_tokens_fall_back_to_model_default(self):
```

Comment:
This test was missing in https://github.com/huggingface/transformers/pull/31254 😛